packages/widgetbook/lib/src/addons/addon.dart

```diff
@@ -34,10 +33,7 @@ class WidgetbookAddOn {
 
   final SingleChildWidget Function(
     BuildContext context,
-    int index,
-  ) providerBuilder;
+  )? providerBuilder;
 
-  final int Function(BuildContext context) selectionCount;
-
-  final String Function(BuildContext context) getQueryParameter;
+  final Map Function(BuildContext context) getQueryParameter;
```

Comment:
This should be of type `Map<String, dynamic>` or of `Map<String, String>` (but I think <String, String> might not be possible)

Reply:
Looks like  `Map<String, String>` works

Reply:
Though i made it `Map<String, dynamic>` to act as a safety net